### PR TITLE
Add new emojis, tidy up `eventstats.py`

### PR DESCRIPTION
**Pull request by mathsman5133**

### CODE REVIEW

File: cogs/eventstats.py [hunk 1]
```diff
@@ -38,34 +61,35 @@ def __init__(self, bot):
                 """
         fetch = await ctx.db.fetch(query, ctx.config.id)
 
-        attacks = {n['player_tag']: n['attacks'] for n in fetch}
-
         table = formatters.CLYTable()
-        title = f"Attack wins for {ctx.config.event_name}"
```
Comment (mathsman5133) [inline cogs/eventstats.py:44]: merge conflicts gone wrong

File: cogs/eventstats.py [hunk 2]
```diff
@@ -38,34 +61,35 @@ def __init__(self, bot):
...
+        )
         await ctx.send(embed=e)
 
     @eventstats.command(name='defenses', aliases=['defense', 'defences', 'defence'])
-    @requires_config('event', invalidate=True)
+    @requires_config('event')
     async def eventstats_defenses(self, ctx):
         """Get defense wins for all clans.
 
-        **Parameters**
```
Comment (mathsman5133) [inline cogs/eventstats.py:60]: add this back in

File: cogs/eventstats.py [hunk 3]
```diff
@@ -38,34 +61,35 @@ def __init__(self, bot):
...
-        :key: Season ID (optional - defaults to last season)
+        **Format**
+        :information_source: `+eventstats defenses`
 
-        **Example**
-        :white_check_mark: `+eventstats defenses`
```
Comment (mathsman5133) [inline cogs/eventstats.py:64]: add these back

File: cogs/eventstats.py [hunk 4]
```diff
@@ -38,34 +61,35 @@ def __init__(self, bot):
...
-        :white_check_mark: `+eventstats defenses 1`
+       **Example**
+       :white_check_mark: `+eventstats defenses`
```
Comment (mathsman5133) [inline cogs/eventstats.py]: get rid of this

File: cogs/eventstats.py [hunk 5]
```diff
@@ -112,56 +143,62 @@ def __init__(self, bot):
...
+  
         **Parameters**
         :key: Season ID (optional - defaults to last season)
 
+        **Format**
+        :information_source: `+eventstats donations SEASON_ID`
+
         **Example**
         :white_check_mark: `+eventstats donations`
-        :white_check_mark: `+eventstats donors 4`
```
Comment (mathsman5133) [inline cogs/eventstats.py:139]: add this back in

**Review by wpmjones — APPROVED** [review]
In looking at it, everything looks good.  By this evening, I should be able to run tests locally (setting up redis on my linux laptop now).  But I think this looks good to go for now.

**Pull request merged**